### DISCUSSION TITLE
Missing something obvious?

### DISCUSSION THREAD

**Discussion by olbasoil** [klaasnicolaas/home-assistant-glow#206] [Q&A] [answered]
Just starting with Glow. I have the physical hardware setup and can visit 'home-assistant-glow' on my ESPHome tab, where I see figures for House-Power consumption and House-Total Energy. But how do I get these in a lovelace card? I can't see any entity or sensor that relates to Glow. The Energy dashboard/ Grid Consumption/Add consumption shows a couple of sensors from smart plugs, but nothing else.

**Comment by klaasnicolaas** [comment]
Suspect that you forgot to link the glow to your Home Assistant via the esphome integration, although HA normally automatically finds all your esphome nodes through auto discovery.

![image](https://user-images.githubusercontent.com/20448157/194711846-6bf0350a-d93a-4c1f-a5dc-84fbcb95427c.png)

**Comment by olbasoil** [comment, marked as answer]
How do I link the glow to Home Assistant? I installed the integration,
added home-assistant-glow.yaml. I have validated, installed and cleaned the
build files. It seems to be working when I go to
http://home-assistant-glow.local/ but there's no sign of anything as an
entity or service?

**Reply by olbasoil** [reply]
It is all working now. I had to add the ESP32 using the  'add integration' button. I had thought that I'd done that as my first step, but I must have deleted it. Thanks for your help.
